File: spec/java/src/io/kaitai/struct/specwrite/TestOpaqueExternalType02Parent.java
package io.kaitai.struct.specwrite;

import io.kaitai.struct.KaitaiStruct.ReadWrite;
import io.kaitai.struct.testwrite.OpaqueExternalType02Parent;
import org.testng.annotations.Test;

public class TestOpaqueExternalType02Parent extends CommonSpec {
    @Override
    protected Class<? extends ReadWrite> getStructClass() {
        return OpaqueExternalType02Parent.class;
    }

    @Override
    protected String getSrcFilename() {
        return "term_strz.bin";
    }
}

File: spec/python/specwrite/test_opaque_external_type_02_parent.py
import unittest
from specwrite.common_spec import CommonSpec

from testwrite.opaque_external_type_02_parent import OpaqueExternalType02Parent

class TestOpaqueExternalType02Parent(CommonSpec.Base):
    def __init__(self, *args, **kwargs):
        super(TestOpaqueExternalType02Parent, self).__init__(*args, **kwargs)
        self.struct_class = OpaqueExternalType02Parent
        self.src_filename = 'src/term_strz.bin'
